src/github-auth.js

```diff
@@ -1,8 +1,32 @@
 const debug = require('debug')('taskcluster-github:github-auth');
-const Github = require('@octokit/rest');
 const Promise = require('bluebird');
 const jwt = require('jsonwebtoken');
 
+const retryPlugin = (octokit, options) => {
+
+  const retries = 7;
+  const baseBackoff = 100;
+  const sleep = timeout => new Promise(resolve => setTimeout(resolve, timeout));
+
+  octokit.hook.wrap('request', async (request, options) => {
+    let response;
+    for (let attempt = 0; attempt < retries; attempt++) {
```

Comment:
I assume this form is used ironically, since the hipsters of last year hated it? :wink:

Reply:
I honestly forgot which order the x;y;z went in! It's been that long since I wrote something like that.

Honestly it just worked better for my mental model of this.